### PR TITLE
Mention custom images for Server Core

### LINKED ISSUE #13409: Server Core for .Net Framework Insufficient
My ticket on the server core project, that .net framework is not full usable with the server core image was confirmed, but closed. So it will never fixed. So please add a hint that server core has only limited .net framework support.
See https://github.com/microsoft/dotnet-framework-docker/issues/299

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 888b22d0-7261-c6fd-e4b3-07a4c1f66d06
* Version Independent ID: ddfcec03-a396-8851-49cd-629b9c7edc8a
* Content: [What OS to target with .NET containers](https://docs.microsoft.com/en-us/dotnet/architecture/microservices/net-core-net-framework-containers/net-container-os-targets)
* Content Source: [docs/architecture/microservices/net-core-net-framework-containers/net-container-os-targets.md](https://github.com/dotnet/docs/blob/master/docs/architecture/microservices/net-core-net-framework-containers/net-container-os-targets.md)
* Product: **dotnet**
* Technology: **dotnet-ebooks**
* GitHub Login: @nishanil
* Microsoft Alias: **nanil**

### PR DESCRIPTION
**Pull request by mvelosop**
## Summary

Mention that you can find incompatibilities when using Server Core and that they can be fixed adding files at image build time.

Fixes https://github.com/dotnet/docs/issues/13409

### CODE REVIEW

File: docs/architecture/microservices/net-core-net-framework-containers/net-container-os-targets.md [hunk 1]
```diff
@@ -20,6 +20,9 @@ In Figure 3-1 you can see the possible OS version depending on the .NET framewor
 
 You can also create your own Docker image in cases where you want to use a different Linux distro or where you want an image with versions not provided by Microsoft. For example, you might create an image with ASP.NET Core running on the traditional .NET Framework and Windows Server Core, which is a not-so-common scenario for Docker.
 
+> [!IMPORTANT]
+> When using Windows Server Core images you might find that some DLLs that are missing, when compared to full Windows images. You might be able to solve the problem creating a custom Server Core image, adding the missing files at image build time.
```
Comment (mairaw) [inline docs/architecture/microservices/net-core-net-framework-containers/net-container-os-targets.md]:
```suggestion
> When using Windows Server Core images, you might find that some DLLs are missing, when compared to full Windows images. You might be able to solve this problem by creating a custom Server Core image, adding the missing files at image build time.
```

File: docs/architecture/microservices/net-core-net-framework-containers/net-container-os-targets.md [hunk 2]
```diff
@@ -29,6 +32,11 @@ When you add the image name to your Dockerfile file, you can select the operatin
 | mcr.microsoft.com/dotnet/core/aspnet:2.2-alpine | .NET Core 2.2 runtime-only on Linux Alpine distro |
 | mcr.microsoft.com/dotnet/core/aspnet:2.2-nanoserver-1803 | .NET Core 2.2 runtime-only on Windows Nano Server (Windows Server version 1803) |
 
+## Additional resources
+
+- **Server Core for .Net Framework Insufficient (GitHub issue)**
```
Comment (mairaw) [inline docs/architecture/microservices/net-core-net-framework-containers/net-container-os-targets.md]:
```suggestion
- **Server Core for .NET Framework Insufficient (GitHub issue)**  
```

File: docs/architecture/microservices/net-core-net-framework-containers/net-container-os-targets.md [hunk 3]
```diff
@@ -29,6 +32,11 @@ When you add the image name to your Dockerfile file, you can select the operatin
 | mcr.microsoft.com/dotnet/core/aspnet:2.2-alpine | .NET Core 2.2 runtime-only on Linux Alpine distro |
 | mcr.microsoft.com/dotnet/core/aspnet:2.2-nanoserver-1803 | .NET Core 2.2 runtime-only on Windows Nano Server (Windows Server version 1803) |
 
+## Additional resources
+
+- **Server Core for .Net Framework Insufficient (GitHub issue)**  
+  <https://github.com/dotnet/docs/issues/13409>
```
Comment (mairaw) [inline docs/architecture/microservices/net-core-net-framework-containers/net-container-os-targets.md]: why list the docs issue as an additional resource?

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @mvelosop. Left some suggestions.

File: docs/architecture/microservices/net-core-net-framework-containers/net-container-os-targets.md (hunk 3 above)
Comment (mairaw) [reply docs/architecture/microservices/net-core-net-framework-containers/net-container-os-targets.md]: The real issue seems to be discussed at https://github.com/microsoft/dotnet-framework-docker/issues/299. Wouldn't that be a better place to link to?

Comment (mvelosop) [reply docs/architecture/microservices/net-core-net-framework-containers/net-container-os-targets.md]: Yep!

**Review by mairaw — APPROVED** [review]

**Pull request merged**